content/articles/using-lottie-animations-in-flutter/index.md

```diff
@@ -0,0 +1,324 @@
+Animations are very important in every mobile application. They make the user interface of an application appealing and fun to use. However, animations may be challenging to implement in applications. That is where Lottie animations come in. 
+
+A Lottie is a JSON-based animation file. They are used both as static assets and network assets. In addition, they polish your app's user interface and make it intuitive to users.
+
+In this article, we will implement Lottie animation in a flutter application. Animation controllers in Flutter give us high-level control over our animations. Therefore, we will be using an animation controller to control the state of our Lottie animations in our Flutter app.
+
+### Prerequisites
+
+1. A solid understanding of the Dart programing language.
+2. A solid understanding of Flutter widgets.
+3. Basic knowledge of using animation controllers in Flutter.
+
+Let us jump right into it!
+
+### Step one: Adding Lottie dependency to your project
+
+- Goto the terminal in your editor and run this command:
+
+```dart
```

Comment:
Inform the user to do this after setting app the application or provide a stepwise process of setting up the application until the point of adding the Lottie dependency.

Reply:
Okay